x-pack/plugin/esql/qa/testFixtures/src/main/resources/stats.csv-spec

```diff
@@ -2018,6 +2018,19 @@ M
 null
 ;
 
+shadowingInternalWithGroup2#[skip:-8.14.1,reason:implemented in 8.14]
```

Comment:
I guess this is not related, but good to add a shadowing test within the groups in particular.

Reply:
This could've tripped due to the logic in the resolution of STATS, which _didn't look_ like it correctly tracks the order of the groups. Turns out it did, because `AttributeMap` uses a `LinkedHashMap` internally; but better to have something to guard against regressions.